python/tidy.py

```diff
@@ -103,12 +103,19 @@ def check_whitespace(idx, line):
 
 def check_by_line(file_name, contents):
     lines = contents.splitlines(True)
+    # import pdb; pdb.set_trace()
```

Comment:
This comment can be removed

(sorry I'm not using reviewable, it's too slow on my tethered connection right now)